lib/db.js

```diff
@@ -908,6 +908,14 @@ module.exports = (
     )
   }
 
+  DB.prototype.verifyTokensWithMethod = function (tokenId, tokenData) {
+    log.trace({op: 'DB.verifyTokensWithMethod', tokenId: tokenId})
+    return this.pool.post(
+      '/tokens/' + tokenId + '/verifyWith',
```

Comment:
Fwiw, I much prefer the template string approach you used below. Should we aim to be consistent for new code?